tests/TestHelpers/WebDavHelper.php

```diff
@@ -46,6 +46,22 @@ class WebDavHelper {
 	 */
 	public static $spacesIdRef = [];
 
+	/**
+	 * clear space id reference for user
+	 *
+	 * @param string|null $user
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public static function removeSpaceIdForUser(
+		?string $user
+	):void {
+		if (\array_key_exists($user, self::$spacesIdRef) && \array_key_exists("personal", self::$spacesIdRef[$user])) {
```

Comment:
no user with duplicate username is pushed this condition should be enough imo :thinking:
```suggestion
		if (\array_key_exists($user, self::$spacesIdRef)) {
```